src/Illuminate/Mail/Markdown.php

```diff
@@ -104,15 +103,13 @@ public function renderText($view, array $data = [])
      */
     public static function parse($text)
     {
-        $environment = Environment::createCommonMarkEnvironment();
-
-        $environment->addExtension(new TableExtension);
-
         $converter = new CommonMarkConverter([
             'allow_unsafe_links' => false,
-        ], $environment);
+        ]);
+
+        $converter->getEnvironment()->addExtension(new TableExtension());
 
-        return new HtmlString($converter->convertToHtml($text));
+        return new HtmlString((string) $converter->convertToHtml($text));
```

Comment:
In `1.x` the `convertToHtml()` method returns a `string`.  In `2.x` it returns a `Stringable` object.  Casting the output to a string allows us to support both.